site/docs/reference/Connectors/materialization-connectors/starburst.md

```diff
@@ -0,0 +1,136 @@
+# Starburst
+
+This connector materializes Flow collections into tables in a Starburst Galaxy.
+
+The connector first uploads data changes to a S3 AWS storage.
+From there, it transactionally applies the changes to the Starburst tables.
+
+[`ghcr.io/estuary/materialize-starburst:dev`](https://ghcr.io/estuary/materialize-starburst:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* A Starburst Galaxy account (To crate one: [Staburst Galaxy start](https://www.starburst.io/platform/starburst-galaxy/start/) that includes:
+  * Cluster with catalog on S3
+  * A schema, to which you'll materialize data
+  * Storage on S3 for temporary data with `awsAccessKeyId` and `awsSecretAccessKey`
+  * A user with a role assigned that grants the appropriate access levels to these resources.
+* At least one Flow collection
+
+### Setup
+
+To get HOST and PORT go to your Cluster -> Connection info
+
+There is also need to grant access to temporary storage (Roles and privileges -> Select specific role -> Privileges -> Add privilege -> Location). "Create schema and table in location" should be selected.
```

Comment:
Add a link towards https://docs.starburst.io/starburst-galaxy/cluster-administration/manage-cluster-access/manage-users-roles-and-tags/account-and-cluster-privileges-and-entities.html#location-privileges-

Add also the fact that the location privileges should correspond to the location of the schema configured for this connector.